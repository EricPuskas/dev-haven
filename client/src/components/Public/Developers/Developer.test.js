import { shallow } from "enzyme";
import React from "react";
import Developer from "./Developer";

it("expect to render Developer component", () => {
  const mockDev = {
    user: {
      _id: 1,
      name: "John",
      avatar: "Something"
    },
    status: "Dev",
    company: "Facebook",
    location: "USA",
    skills: [1, 2, 3, 4, 5]
  };
  expect(shallow(<Developer profile={mockDev} />).debug()).toMatchSnapshot();
});
