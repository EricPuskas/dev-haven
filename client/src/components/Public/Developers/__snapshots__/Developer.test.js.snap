// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Developer component 1`] = `
"<div className=\\"profile bg-light\\">
  <img src=\\"Something\\" alt=\\"Avatar\\" className=\\"round-img\\" />
  <div>
    <h2>
      John
    </h2>
    <p>
      Dev
       
      <span>
         at 
        Facebook
      </span>
    </p>
    <p className=\\"my-1\\">
      <span>
        USA
      </span>
    </p>
    <Link to=\\"/developers/1\\" className=\\"btn btn-red\\">
      View Profile
    </Link>
  </div>
  <ul>
    <li className=\\"text-primary\\">
      <i className=\\"fas fa-check\\" />
       
      1
    </li>
    <li className=\\"text-primary\\">
      <i className=\\"fas fa-check\\" />
       
      2
    </li>
    <li className=\\"text-primary\\">
      <i className=\\"fas fa-check\\" />
       
      3
    </li>
    <li className=\\"text-primary\\">
      <i className=\\"fas fa-check\\" />
       
      4
    </li>
  </ul>
</div>"
`;
